packages/test/test-service-load/src/loadTestDataStore.ts

```diff
@@ -573,27 +574,18 @@ class LoadTestDataStore extends DataObject implements ILoadTest {
 			config.testConfig.useVariableOpSize === true
 				? Math.floor(Math.random() * opSizeinBytes)
 				: opSizeinBytes;
+		const largeOpRate = config.testConfig.largeOpRate ?? 1;
+		let opsSent = 0;
 
-		const sendSingleOp =
-			opSizeinBytes === 0
-				? () => {
-						dataModel.counter.increment(1);
-				  }
-				: () => {
-						const opPayload = generateContentOfSize(getOpSizeInBytes());
-						const opKey = Math.random().toString();
-						dataModel.sharedmap.set(opKey, opPayload);
-				  };
+		const sendSingleOp = () => {
+			if (opSizeinBytes > 0 && largeOpRate > 0 && opsSent % largeOpRate === 1) {
+				dataModel.sharedmap.set(uuid(), generateContentOfSize(getOpSizeInBytes()));
```

Comment:
do we actually want a random key? this will cause each large op to increase to total payload size for snapshots, which seems orthogonal to this change. ideally we could run stress forever, and not have it fail, but this grows unbound so will eventually get too large.